schema/me/save_artwork.js

```diff
@@ -0,0 +1,28 @@
+import gravity from '../../lib/loaders/gravity';
+import { GraphQLString, GraphQLBoolean } from 'graphql';
+import { mutationWithClientMutationId } from 'graphql-relay';
+import { ArtworkType } from '../artwork/index';
+
+export default mutationWithClientMutationId({
+  name: 'SaveArtwork',
+  decription: 'Save (or remove) an artwork to (from) a users default collection.',
+  inputFields: {
+    artwork_id: {
+      type: GraphQLString,
+    },
+    remove: {
+      type: GraphQLBoolean,
+    },
+  },
+  outputFields: ArtworkType.getFields(),
```

Comment:
ooo i didnt know you could do `getFields`. I had previously moved the fields out of the type struct (https://github.com/artsy/metaphysics/blob/20b68bc7e5f259408c123b6331b4f7eefe749568/schema/me/conversations.js#L42-L90) b/c this expects just the fields and I didn't know how to get to the fields from the type.